src/Datadog.Trace/FrameworkDescription.NetCore.cs

```diff
@@ -8,8 +8,15 @@ namespace Datadog.Trace
 {
     internal partial class FrameworkDescription
     {
+        private static FrameworkDescription _instance;
+
         public static FrameworkDescription Create()
         {
+            if (_instance != null)
```

Comment:
If we're going to cache this, we should probably rename the `Create()` method since it longer creates an instance on every call. We could change it into an `Instance` property like we do with other singletons.

Reply:
Separated into https://github.com/DataDog/dd-trace-dotnet/pull/1092

Reply:
Thanks.

Reply:
I just hadn't pushed yet when you looked again, my apologies.

Reply:
Oops, sorry `:fastpress:`